bootstrap/pre-pivot.sh

```diff
@@ -25,6 +25,12 @@ MACHINE_CONFIG_OSCONTENT=$(image_for machine-os-content)
 echo "${MACHINE_CONFIG_OSCONTENT}" > /etc/pivot/image-pullspec
 touch /run/pivot/reboot-needed
 
+# Update bootstrap's kernel args
+echo -e "DELETE mitigations=auto,nosmt" > /etc/pivot/kernel-args
+echo -e "DELETE systemd.unified_cgroup_hierarchy=0" >> /etc/pivot/kernel-args
+echo -e "ADD cgroup_no_v1=all" >> /etc/pivot/kernel-args
+echo -e "ADD psi=1" /etc/pivot/kernel-args
```

Comment:
side note: We should be able to do this with an ignition config now: https://github.com/coreos/fedora-coreos-config/blob/testing-devel/tests/kola/ignition/kargs/config.ign

Reply:
Right, the right way would be setting those via installer anyway - but I'd rather have our fork merged first